modules/tinymce/src/plugins/table/test/ts/module/test/TableTestUtils.ts

```diff
@@ -239,6 +239,16 @@ const createRow = (cellContents: string[]): SugarElement<HTMLTableRowElement> =>
   return tr;
 };
 
+/** Opens the specified properties dialog, selects the class called `classTitle`, confirms changes and submits dialog. */
```

Comment:
I think we should try to avoid comments like this if we can if the code is not clear enough then break things out as named functions that document the code instead. For example the execCommand and pWaitForDialog could be one function called `pOpenDialogByCommand(editor, cmd)` then the click on select and wait and click on item could be `pSelectListBoxValue(editor, classTitle)` then the close thing could be `pSubmitDialog(editor)` or something.